src/lib389/pyproject.toml

```
@@ -0,0 +1,95 @@
+[build-system]
+requires = [
+    "setuptools",
+    "wheel",
+    "argparse-manpage[setuptools]",
+    "pyasn1",
+    "pyasn1-modules",
+    "python-dateutil",
+    "argcomplete",
+    "python-ldap",
+    "distro",
+    "cryptography",
+    "psutil",
+    "pytest",
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "lib389"
+version = "3.1.2"  # Should match the main 389-ds-base version
+description = "A library for accessing, testing, and configuring the 389 Directory Server"
+readme = "README.md"
+license = "GPL-3.0-or-later"
+license-files = ["LICENSE"]
```

```suggestion
license = {text = "GPL-3.0-or-later"}
```
license-files is supported by setuptools-77.0.3 and higher: https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license-and-license-files
In Fedora 42 we have 74, so this won't work.

---

Careful about the gpl file missing from the rpm package. See my comment in the general comments section.

---

You could add it via `[tool.setuptools]` `license-files`. 

```toml
[tool.setuptools]
license-files = ["LICENSE*"]
```

This is now deprecated in new setuptools (in favor of the project table), but still works.